### PR TITLE
[WIP] Update README.md

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,14 +31,19 @@ can export before running the tests:
...
+- `RESINOS_TESTS_RESINOS_VERSION`: the device version to test (default is `latest`)
+- `RESINOS_TESTS_RESINOS_VERSION_UPDATE`: the version that the device will be updated to (default is `latest`)
 - `RESINOS_TESTS_EMAIL`: the email of the test account
 - `RESINOS_TESTS_PASSWORD`: the password of the test account
-- `RESINOS_TESTS_WIFI_SSID`: the wifi SSID the provisioned device should
-  connect to
-- `RESINOS_TESTS_WIFI_KEY`: the wifi key the provisioned device should use
-- `RESINOS_TESTS_DISK`: the device that should be provisioned with resinOS
-  (i.e. `/dev/disk3`)
+- `RESINOS_TESTS_NETWORK`: choose the network connection type (`ethernet` or `wifi+ethernet`)
```
Comment (horia-delicoti) [inline README.md]: Actually, we are verifying if the env variable is `ethernet` here -> https://github.com/resin-io/resinos-tests/blob/master/lib/components/resinos/default.js#L40
Anything else besides `ethernet` is also configuring wifi.

**Comment by telphan** [comment]
I am marking this as WIP and we will amend to it until we release a stable version.

**Comment by horia-delicoti** [comment]
Sure

**Comment by telphan** [comment]
Fixed here https://github.com/balena-io/leviathan/pull/288

**Pull request closed without merging**